app/forms/declaration/NactCode.scala

```diff
@@ -33,7 +33,7 @@ case class NactCode(nactCode: String) extends Ordered[NactCode] {
 object NactCode extends DeclarationPage with FieldMapping {
   implicit val format = Json.format[NactCode]
 
-  val pointer: ExportsFieldPointer = CommodityDetails.combinedNomenclatureCodePointer
+  val pointer: ExportsFieldPointer = "nactCode"
```

Comment:
When the pointers get produced for this and taric, will they have a sequenceId?  Because if we are deciding to point at the parent, we won't want that.

Reply:
`ExportsPointerToWCOPointer` throws an exception when there are mismatching quantities of $ sequence placeholders either side of a mapping.

Reply:
Yes they will have sequenceId. We will have to make the BE handle these sequenceId (ignore them) because we want to keep this sequenceId data in the FE for other functionality (like showing users the difference between two declaration versions). :+1: